SteakBot.Core/EventHandlers/ReactionEventHandler.cs

```diff
@@ -11,8 +13,12 @@ public async Task HandleReactionAddedAsync(Cacheable<IUserMessage, ulong> arg1,
         {
             var channel = arg3.Channel;
             var message = await channel.GetMessageAsync(arg3.MessageId);
-            var messageJumpUrl = message.GetJumpUrl();
+            if (await ShouldSkipReaction(channel, message, 10))
```

Comment:
The message count should not be hardcoded.

Reply:
Agreed and resolved.